test/uk/gov/hmrc/transitmovementsrouter/generators/TestModelGenerators.scala

```diff
@@ -123,4 +127,25 @@ trait TestModelGenerators extends BaseGenerators {
     )
   }
 
+  implicit def arbitrarySdesResponse(conversationId: ConversationId): Arbitrary[SdesNotificationItem] = Arbitrary {
+    for {
+      filename          <- Gen.alphaNumStr
+      correlationId     <- Gen.alphaNumStr
+      checksum          <- Gen.stringOfN(4, Gen.alphaChar)
+      checksumAlgorithm <- Gen.alphaNumStr
+      notification      <- Gen.oneOf(SdesNotification.values)
+      received   = Instant.now()
+      properties = Seq(SdesProperties(RouterHeaderNames.CONVERSATION_ID, conversationId.value.toString))
```

Comment:
better to create conversationId gen here with this gen instead of creating and passing it in different test cases as parameters.

Reply:
I don't see a problem over here as the conversationId using should match with movementId and messageId